auto_backup/models/db_backup.py

```diff
@@ -180,8 +181,8 @@ def action_backup(self):
 
                             # Copy cached backup to remote server
                             with remote.open(
-                                    os.path.join(rec.folder, filename),
-                                    "wb") as destiny:
+                                os.path.join(rec.folder, filename), "wb"
```

Comment:
Looks like this isn't changing anything, can you remove it from the commit?

Reply:
Removed